### DISCUSSION TITLE
Using teavm with ToneJs

### DISCUSSION THREAD

**Discussion by ThanosFisherman** [konsoletyper/teavm#933] [Q&A]
How can I use [ToneJs](https://github.com/Tonejs/Tone.js) in teavm? I would like to be able to call something along the lines of

```javascript
    if (Tone.context.state !== "running") {
        Tone.start();
      }
      const synth = new Tone.Synth().toDestination();
      synth.triggerAttackRelease("C4", "8n");
```

Through Java/Kotlin code. My main issue is poiting teavm to use Tone.js module.

**Comment by konsoletyper** [comment]
You should read this first: https://teavm.org/docs/runtime/jso.html
